service/src/test/java/com/myhome/controllers/HouseControllerTest.java

```diff
@@ -48,25 +58,42 @@
 import static org.mockito.BDDMockito.given;
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
 
 class HouseControllerTest {
 
   private final String TEST_HOUSE_ID = "test-house-id";
   private final String TEST_MEMBER_ID = "test-member-id";
-
   private final int TEST_HOUSES_COUNT = 2;
   private final int TEST_HOUSE_MEMBERS_COUNT = 2;
 
+  private static final String COMMUNITY_ADMIN_ID = "1";
+  private static final String COMMUNITY_ADMIN_NAME = "Test Name";
+  private static final String COMMUNITY_ADMIN_EMAIL = "testadmin@myhome.com";
+  private static final String COMMUNITY_ADMIN_PASSWORD = "testpassword@myhome.com";
+  private static final String COMMUNITY_HOUSE_ID = "2";
+  private static final String COMMUNITY_HOUSE_NAME = "Test House";
+  private static final String COMMUNITY_NAME = "Test Community";
+  private static final String COMMUNITY_ID = "3";
+  private static final String COMMUNITY_DISTRICT = "Wonderland";
+
+  @InjectMocks
+  private CommunityController communityController;
```

Comment:
why `HouseControllerTest` has InjectMocks on `CommunityController` and `HouseController`? Should be only `HouseController`